src/components/graphs/DataByHourGraph/index.tsx

```diff
@@ -118,26 +96,161 @@ function DataByHourGraph({ range, stats = [] }: Props) {
+            const totalBytes = stat.bytes_to
+                ? stat.bytes_to + stat.bytes_by
+                : stat.bytes_by;
+
+            return {
+                docs: totalDocs,
+                bytes: totalBytes,
+                timestamp: stat.ts,
+            };
+        });
+    }, [stats]);
+
     // Set the main bulk of the options for the chart
     useEffect(() => {
+        // Function to format that handles both dimensions. This allows the tooltip
+        //  formatter to not worry about dimensions and just pass them in here
+        const formatter = (value: any, dimension: 'bytes' | 'docs') => {
+            if (!Number.isInteger(value)) {
+                return intl.formatMessage({
+                    id: 'common.missing',
+                });
+            }
+
+            if (dimension === 'docs') {
+                return readable(value, 2, false);
+            }
+
+            return defaultDataFormat(value);
+        };
+
+        const bytesSeries: EChartsOption['series'] = {
+            barMinHeight: 1,
+            encode: {
+                x: 'timestamp',
+                y: 'bytes',
+            },
+            markLine: {
+                data: [{ type: 'max', name: 'Max' }],
+                label: {
+                    backgroundColor: eChartsColors[0],
+                    color: 'white',
+                    padding: 3,
+                    position: 'start',
+                    formatter: ({ value }: any) => {
+                        return formatter(value, 'bytes');
+                    },
+                },
+                symbolSize: 0,
+            },
+            name: intl.formatMessage({ id: 'data.data' }),
+            type: 'bar',
+            yAxisIndex: 0,
+        };
+
+        const docsSeries: EChartsOption['series'] = {
+            barMinHeight: 1,
+            encode: {
+                x: 'timestamp',
+                y: 'docs',
+            },
+            markLine: {
+                data: [{ type: 'max', name: 'Max' }],
+                label: {
+                    backgroundColor: eChartsColors[1],
+                    color: 'black',
+                    padding: 3,
+
+                    position: 'end',
+                    formatter: ({ value }: any) => {
+                        return formatter(value, 'docs');
+                    },
+                },
+                symbolSize: 0,
+            },
+            name: intl.formatMessage({ id: 'data.docs' }),
+            type: 'bar',
+            yAxisIndex: 1,
+        };
+
         const option: EChartsOption = {
             animation: false,
             darkMode: theme.palette.mode === 'dark',
             legend: legendConfig,
+            series: [bytesSeries, docsSeries],
+            useUTC: true,
+            // Setting dataset here because setting in a stand alone set option cause the chart to go blank
+            dataset: {
+                dimensions: ['timestamp', 'bytes', 'docs'],
```

Comment:
These strings are repeated throughout the file. Could we store them off in a constant for ease of reference (and maintenance)?